app/sqaws.py

```diff
@@ -108,8 +110,15 @@ def build_instance_model(pricing: PricingData, region_name: str, instance_dict:
     monthly_storage_price = estimate_monthly_ebs_storage_price(region_name, instance_dict['InstanceId'])
     monthly_price = (monthly_server_price + monthly_storage_price) if state == 'running' else monthly_storage_price
 
-    stop_after = tags.get('Stop after', tags.get('Stop After', tags.get('StopAfter', '')))
-    terminate_after = tags.get('Terminate after', tags.get('Terminate After', tags.get('TerminateAfter', '')))
+    stop_after_str, terminate_after_str = get_stop_and_terminate_after(tags)
+    if stop_after_str == '':
```

Comment:
With the change for a default tag name I suggested in another comment, I belive these if/else statements could be `stop_after = tags.get(stop_after_str, "")`